src/site-type/html.php

```diff
@@ -116,13 +124,13 @@ public function create( $args, $assoc_args ) {
 		$this->skip_status_check                   = \EE\Utils\get_flag_value( $assoc_args, 'skip-status-check' );
 		$this->site_data['site_container_fs_path'] = get_public_dir( $assoc_args );
 
-		$this->site_data['site_ssl'] = get_value_if_flag_isset( $assoc_args, 'ssl', [ 'le', 'self', 'inherit' ], 'le' );
+		$this->site_data['site_ssl'] = get_value_if_flag_isset( $assoc_args, 'ssl', [ 'le', 'self', 'inherit', 'custom' ], 'le' );
```

Comment:
If it is custom ssl then take the key and crt path here itself and validate if they are present. If not then throw exception from here itself.

To validate, 
1. check if the file itself is present in the path given by user.
2. check if the file is present from the path relative to PWD from where the site creation command is being executed.

If this process is being repeated in all three site types then create a file validation function in site-utils and move the code for it there and use it here and in other site types.